### PR TITLE
OGC API - Processes / REST API docs + some fixes

### PR DESCRIPTION
**Pull request by idanmiara**
# Overview

OGC API - Processes / REST API - Documentation and some fixes.

# Related Issue / Discussion

https://github.com/geopython/pywps/pull/588

# Contribution Agreement

(as per https://github.com/geopython/pywps/blob/master/CONTRIBUTING.rst#contributions-and-licensing)

- [X] I'd like to contribute [feature X|bugfix Y|docs|something else] to PyWPS. I confirm that my contributions to PyWPS will be compatible with the PyWPS license guidelines at the time of contribution.
- [X] I have already previously agreed to the PyWPS Contributions and Licensing Guidelines

### CODE REVIEW

**Comment by idanmiara** [comment]
@cehbrecht Is anything else is needed for this PR and for releasing v4.5?

**Review by cehbrecht — APPROVED** [review]

File: docs/api_rest.rst [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+This standard builds on the OGC Web Processing Service (WPS) 2.0 Standard
+and defines the processing interface to communicate over a RESTful protocol using JSON encodings.
+
+For more details about the standard please refer to https://github.com/opengeospatial/ogcapi-processes
+
+Defining the input/output format (JSON or XML)
+================================================
+
+WPS 1.0 standard defines input and outputs in XML format.
+WPS 2.0 standard defines input and outputs in JSON format.
```
Comment (cehbrecht) [inline docs/api_rest.rst:18]: I would not mention WPS 2.0 (that is still xml based ... not ogcapi - processes). 

* WPS 1.0.0: xml
* WPS 2.0.0: xml (https://docs.opengeospatial.org/is/14-065/14-065.html)
* OGC API - Processes: rest-api, json

**Comment by cehbrecht** [comment]
@idanmiara Thanks for updating the docs :) I have tested it with [emu](https://github.com/bird-house/emu) and it works :)

**Comment by cehbrecht** [comment]
I would skip wps 2.0.0 in the docs ... this is not ogcapi-processes. When updated I will merge this PR and make a new releases 4.5.0.

File: docs/api_rest.rst (hunk 1 above)
Comment (idanmiara) [reply docs/api_rest.rst]: OK, done, thanks!

**Comment by idanmiara** [comment]
Thanks @cehbrecht !

**Comment by idanmiara** [comment]
@cehbrecht Are you releasing v4.5 to pypi or is there anything else needed?

**Comment by cehbrecht** [comment]
@idanmiara There is an ogc coding sprint for ogc-api processes starting tomorrow:

**Comment by idanmiara** [comment]
> @idanmiara There is an ogc coding sprint for ogc-api processes starting tomorrow:
> https://www.ogc.org/ogcevents/ogc-api-virtual-sprint-processes-coverages-records

Thanks for this pointer!
I've registered for the event :)

**Pull request merged**